### ISSUE TITLE
Trading document

### ISSUE THREAD

**Issue by SirLich** [SirLich/technical-bedrock#5]
**Description:** This document should explain how to create trades, with an emphasis on where to place files, and where to find info on the format, rather than explaining in detail the actual stuff in the trade.

**Section:** Concepts

**Sample structure:**
# Trading in Minecraft
Trading is accomplished through the bla component...

Here is where to put the files.

## Trade Types
There are two types
### Economy Trade 
bla
### Trades (old)
bla
## Where to find documentation
bla
## Sample trade file
bla

**Comment by SirLich** [comment]
When doing trade tables, what is the difference of a json in the trading folder as opposed to the one in the trading -> economy trades folder? Do you need both and what is each one for?
nicholasToday at 10:16 PM
I have the same question as well, im not really sure how they differ. But then again, ive never bothered to work with trade tables.
SirLichToday at 10:24 PM
They have a different format.
Trade tables are "simpler"
And are considered the old, outdated format.
PepijnToday at 10:24 PM
In short, trade tables were made for the old villagers
SirLichToday at 10:24 PM
The new economy trade tables represent the new "economy" of the villagers.
So there are tiers, experience levels, they can be recharged by doing tasks, etc.

**Comment by KaiFireborn** [comment]
The Guide, currently being migrated to the wiki, has a Trade Table tutorial. A duplicate is not needed.
